nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -13,8 +13,8 @@ let
     vhostConfig // {
       inherit serverName;
     } // (optionalAttrs vhostConfig.enableACME {
-      sslCertificate = "/var/lib/acme/${serverName}/fullchain.pem";
-      sslCertificateKey = "/var/lib/acme/${serverName}/key.pem";
+      sslCertificate = "${cfg.acmeDir}/${serverName}/fullchain.pem";
```

Comment:
I haven't tested this, but I think you should be able to use `config.security.acme.directory` directly here instead of making an nginx-specific wrapper option.

Reply:
Yes, it appears that this does work - just pushed a commit to use that instead.  Thanks for the suggestion! 😀